ui/tests/acceptance/oidc-config/clients-test.js

```diff
@@ -41,7 +41,16 @@ module('Acceptance | oidc-config/clients', function (hooks) {
 
   test('it renders empty state when no clients are configured', async function (assert) {
     assert.expect(5);
-    this.server.get('/identity/oidc/client', () => overrideMirageResponse(200));
+
+    // * ENSURE CLEAN TEST STATE
+    try {
+      const model = await this.store.findRecord('oidc/client', 'some-app');
+      model.destroyRecord();
```

Comment:
Reading the Ember API docs on [destroyRecord](https://api.emberjs.com/ember-data/release/classes/Model/methods/destroyRecord?anchor=destroyRecord) it looks like it returns a promise, so maybe put an await in front of this as well and then you wouldn't need the settled? Not 100% sure, but just a thought.

Reply:
ah - you know. i think using a `.then` here would be cleanest and ensure the promises chain and are handled correctly!